app/graph/mutations/resend_cancel_invitation_mutation.rb

```diff
@@ -1,28 +1,28 @@
-ResendCancelInvitationMutation = GraphQL::Relay::Mutation.define do
-  name 'ResendCancelInvitation'
+class ResendCancelInvitationMutation < Mutations::BaseMutation
+  graphql_name "ResendCancelInvitation"
 
-  input_field :email, !types.String
+  argument :email, String, required: true
 
-  input_field :action, !types.String
+  argument :action, String, required: true
 
-  return_field :success, types.Boolean
+  field :success, Boolean, null: true
 
-  return_field :team, TeamType
+  field :team, TeamType, null: true
 
-  resolve -> (_root, inputs, _ctx) {
+  def resolve(**inputs)
     user = User.find_user_by_email(inputs[:email])
     if user.nil?
       raise ActiveRecord::RecordNotFound
     else
       case inputs[:action]
-      when 'cancel'
+      when "cancel"
```

Comment:
I didn't intentionally change this, but I've been using the Shopify Ruby LSP for linting and it has this preference. Here are the docs: https://github.com/Shopify/ruby-lsp. It's available as a plugin for VSCode

My understanding of the logic is just use the more flexible quotation rather than tying quote type to whether it interpolates or not, but I'm not sure of the exact justification.